pytentiostat/config_reader.py

```diff
@@ -44,6 +44,8 @@ def parse_config_file(configlocation=None):
 def get_output_params(config_data, override_ts=None):
     data_out_name = config_data["general_parameters"]["data_output_filename"]
     data_out_path = config_data["general_parameters"]["data_output_path"]
+    data_out_name = str(data_out_name)
```

Comment:
nice @aplymill7 , so the writing of the tests has led to more robust code?

Reply:
It has! Makes you think of stuff to try to make hold up to scrutiny...